examples/Example/src/prot_siblings/index.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import {
+  GestureMonitor,
+  useGesture,
+  Gesture,
+} from 'react-native-gesture-handler';
+import { useState } from 'react';
+
+function getState(s: number) {
+  switch (s) {
+    case 0:
+      return 'Undetermined';
+    case 1:
+      return 'Failed';
+    case 2:
+      return 'Began';
+    case 3:
+      return 'Cancelled';
+    case 4:
+      return 'Active';
+    case 5:
+      return 'End';
+  }
+  return s;
+}
+
+function Box(props) {
+  const [s, setS] = useState(0);
+
+  return (
+    <View
+      style={[
+        styles.box,
+        { backgroundColor: props.color },
+        props.overlap ? styles.overlap : {},
+        props.style,
+      ]}></View>
```

Comment:
```suggestion
      ]} />
```

Reply:
Fixed in 8a6a3ba090e2d803bc8c2e4877b6d2c9f6b60412